addons/account/models/account_tax.py

```diff
@@ -288,6 +302,28 @@ def _hook_compute_is_used(self, tax_to_compute):
         '''
         return set()
 
+    def _get_domestic_fiscal_positions(self, country_codes: list):
+        # this doesn't cover all cases as some l10ns don't use the default (consider using the 1st fiscal position in the company instead)
+        return {
+            cc: self.env['account.chart.template'].ref(f"l10n_{cc.lower()}_domestic_fiscal_position", raise_if_not_found=False) or self.env['account.fiscal.position']
```

Comment:
Don't use a `ref`, it should be configurable easily even without using a predefined chart of accounts